DSCResources/MSFT_SecurityOption/MSFT_SecurityOption.psm1

```diff
@@ -224,18 +237,22 @@ function Set-TargetResource
         $Interactive_logon_Display_user_information_when_the_session_is_locked,
 
         [Parameter()]
+        [ValidateSet("Enabled", "Disabled")]
         [System.String]
         $Interactive_logon_Do_not_display_last_user_name,
 
         [Parameter()]
+        [ValidateSet("Enabled", "Disabled")]
         [System.String]
         $Interactive_logon_Do_not_require_CTRL_ALT_DEL,
 
         [Parameter()]
+        [ValidateRange(4, 999)]

```

Comment:
from [this link](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/interactive-logon-machine-account-lockout-threshold) it seems that not allowing 0 is actually limiting the options:
> Values from 1 to 3 are interpreted as 4. 
>
>If you **set the value to 0**, or leave blank, the computer or device will never be locked as a result of this policy setting.